### DISCUSSION TITLE
Question RE: KernelCCA Eigenvalues

### DISCUSSION THREAD

**Discussion by accentuate-the-positive** [deeptime-ml/deeptime#225] [Q&A]
Apologies if this is a stupid question but can you use the eigenvalues output of KernelCCA as a predictor in forecasting model?

**Comment by clonker** [comment]
Hey not a stupid question at all! You _can_ use it for propagation into the future but only in the latent space (meaning the projection space after application of the eigenfunctions). It becomes more tricky when you want to relate it this back to the input space, though. In general it also only will propagate the slow degrees of motion, everything fast (stochastic vibrations and the likes) are projected away. There are Koopman methods which do very well on forecasting (for example this [https://arxiv.org/abs/2004.00574](https://arxiv.org/abs/2004.00574)), but although planned, it is not yet implemented in deeptime. But I think there already is an implementation on github if you want to give it a go.
